src/features/authorization/managers/registerManager.ts

```diff
@@ -24,10 +24,13 @@ export const useRegisterManager = () => {
     email: Yup.string()
       .email(t('signup:incorrectEmail'))
       .required(t('signup:required')),
-    password: Yup.string()
-      .min(8, t('signup:passwordLengthMin'))
-      .max(64, t('signup:passwordLengthMax'))
-      .required(t('signup:required')),
+    password: Yup.string()ß
+      .min(8, 'Password must be at least 8 characters long')
```

Comment:
All validation messages should be taken from translation file.

Reply:
Awesome will go back and fix this ! Thanks for the feedback !